osf/models/institution.py

```diff
@@ -160,24 +161,23 @@ def registrations_relationship_url(self):
 
     @property
     def logo_path(self):
-        if self.logo_name:
-            return '/static/img/institutions/shields/{}'.format(self.logo_name)
-        else:
+        try:
+            return self.asset_files.get(name='logo').file.url
+        except InstitutionAssetFile.DoesNotExist:
             return None
```

Comment:
(Out-of-scope) A note for future improvement, return a placeholder image if missing.